src/assets.rs

```diff
@@ -226,33 +222,71 @@ impl<'a> DerefMut for AssetEditor<'a> {
 
 /// Assets stored in an assets store.
 pub struct Assets {
+    /// A key-value store for assets.
     pub store: AssetsStore,
+
+    /// A set of asset codes loaded from verified asset libraries.
+    pub verified_assets: HashSet<AssetCode>,
 }
 
 impl Assets {
     #![allow(dead_code)]
 
-    /// Create assets.
+    /// Create unverified assets.
+    ///
+    /// To load verified assets afterwards, use `verify_asset`.
     pub fn new<L: Ledger>(store: AssetsStore) -> Result<Self, KeystoreError<L>> {
-        Ok(Self { store })
+        Ok(Self {
+            store,
+            verified_assets: HashSet::new(),
+        })
     }
 
     /// Iterate through the assets.
     pub fn iter(&self) -> impl Iterator<Item = Asset> + '_ {
         self.store.iter().cloned()
     }
 
-    /// Get the asset by the code.
+    /// Load a verified asset library with its trusted signer.
+    ///
+    /// Call this function at least once after assets resource is created.
```

Comment:
We should probably add a caveat here making clear that this function will not affect assets that have already been loaded into memory (e.g. any `Asset` or `AssetEditor` objects you have around). I guess you can't have an `AssetEditor` because it requires `&mut self` but you could have an `Asset` that you previously loaded, and you would need to reload it before its `verified` flag was updated.

Reply:
I think `_before_ its verified flag was updated` should be `_after_ its verified flag is updated` so that we can set a previously unverified asset as verified if it's in the newly-loaded asset library? Am I misunderstanding what you meant?

Reply:
I think we're saying the same thing. What I meant by "before" is the `verified` flag on the `Asset` will not be updated until you reload the asset.